integration-guides/geotab.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Geotab Australia
+---
+
+To connect your Geotab telematics device integration with PathStack, you will need to retrieve your MyGeotab user email, the password you use to log into your MyGeotab account and the database name. 
+
+To find these credentials follow these next steps:
+
+<Steps>
+
+  <Step title = "Create a Geotab Service Account for PathStack">
+  Log in to MyGeotab account and navigate to (1) Administration > (2) Users, and click the (3) Add button. 
+  <img src="images/Integration-guides/geo_1_1.png" />
+  </Step>
+
+  <Step title="Create a new user for PathStack">
+   On the new user page, complete the following:
+
+       <Steps>
+        <Step title = "User (Email) - Email used as PathStack Username">
+        </Step>
+        <Step title = "Password - Create a strong password for one-time user">
+        </Step>
+        <Step title = "Force password change on next login - Toggle to Yes">
+        </Step>
+        <Step title = "Security Clearence - Choose View Only">
+        </Step>
+        <Step title = "Data Access - Select Everything">
+        </Step>
+        <Step title = "Save">
+        </Step>
+        </Steps>
+
+<img src="images/Integration-guides/geo_1.png" />
+
+User (Email) is the username that will be used in PathStack as the credentials Username.
+
+<img src="images/Integration-guides/geo_2.png" />
+
+  </Step>
+  <Step title="Retrieve Database name">
+Once logged into your MyGeotab account, you will be able to retrieve your database name from the search bar at the top of your page. 
+
+The database name sits in the search bar as my.geotab.com/DatabaseName/#dashboard. For example, in the image below the database name would be Nuonic. 
```

Comment:
surround my.geotab.com/DatabaseName/#dashboard with ` character to make it stand out